### DISCUSSION TITLE
RTK 2.0

### DISCUSSION THREAD

**Discussion by Leeasdw** [reduxjs/redux-toolkit#4513] [Q&A]
Hello, it is  strictly required to upgrade to latest reaсt аnd rеduх in order to move to RTK  2.0?

**Comment by markerikson** [comment]
Not _strictly_ required, but you'd need to enforce the peer dep differences, and there may be some types issues as well.

Redux Toolkit 2.0 doesn't _need_ React-Redux v9 and React 18, but we did make a lot of changes around peer dependencies and types. Things would still _run_ okay otherwise, but you may have to do some config work to make things compile okay.
